client/my-sites/customer-home/cards/tasks/site-setup-list/index.jsx

```diff
@@ -61,7 +62,6 @@ const startTask = ( dispatch, task, siteId ) => {
 
 const skipTask = ( dispatch, task, siteId ) => {
 	dispatch( requestSiteChecklistTaskUpdate( siteId, task.id ) );
-	dispatch( requestHomeLayout( siteId ) );
```

Comment:
This has been moved to a separate `useEffect` below in order to handle more edge cases where site setup is complete after skipping a task (i.e. create site menu).